doc/en/md/09-Advanced_use.md

```diff
@@ -318,7 +318,8 @@ The data will remain unchanged until you ask for a calculation of the travel tim
 
 On the left, a vertical bar of buttons allows actions on this map.
 
-* ![](./images/Map/marker.png) : Adds an start mark to the selected location
+* ![](./images/Map/center.png) : Centers the track on the map

```

Comment:
@patrovite Shouldn't we use ../images instead of ./images ? 
It seems to me that it would avoid duplicating the folders ./fr/images and ./en/images